### PR TITLE
7791 integration test of liquidation flow 2b

### LINKED ISSUE #7791: test of liquidations after upgrade
## What is the Problem Being Solved?

Auction and liquidation require the coordination of multiple contracts with precise timing. We need to be confident these work after upgrade. We want these in CI to prevent breakage.

## Description of the Design

Ideally we get this into the Docker upgrade-test, but the main bootstrap configures auction StartFrequency to 60min. Two auctions get scheduled at start so that's 2 hrs, which is too long for CI.

One alternative is to test this only in the "test" bootstrap mode and set its config to a rapid auction StartFrequency.

Another is to develop confidence that a bootstrapTest would suffice and implement it there instead.

### PR DESCRIPTION
**Pull request by turadg**
closes: #7791

## Description

The manual liquidation testing had a "reconstitution" flow, 2b. This adds that to the bootstrap tests. With two such similar tests, it also refactors to DRY out some of the logic.

### Security Considerations

n/a

### Scaling Considerations

n/a
### Documentation Considerations

n/a
### Testing Considerations

!

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
looks like [the failure in test-cosmic-swingset (16.x)](https://github.com/Agoric/agoric-sdk/actions/runs/5095986417/jobs/9161491132?pr=7857) might be a flake. I'll re-run it.

File: packages/vats/test/bootstrapTests/test-liquidation-1.js [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    /** @type {import('@agoric/inter-protocol/src/auction/scheduler.js').ScheduleNotification} */
+    const liveSchedule = readLatest('published.auction.schedule');
+    t.is(liveSchedule.activeStartTime, null);
+    t.like(readLatest('published.vaultFactory.managers.manager0.metrics'), {
+      numActiveVaults: setup.vaults.length,
+      numLiquidatingVaults: 0,
+    });
+
+    // advance time to start an auction
+    console.log('step 0 of 10');
```
Comment (Chris-Hibbert) [inline packages/vats/test/bootstrapTests/test-liquidation-1.js:253]: Can these `console.log()` messages be dropped or converted to inactive `trace()`? I can see how they'd be helpful while building and debugging the test, but their pretty noisy after that.

File: packages/vats/test/bootstrapTests/test-liquidation.js [hunk 2]
```diff
@@ -536,23 +537,23 @@ test.serial('scenario: Flow 1', async t => {
...
-    shortfallBalance: { value: scale6(outcome.reserve.shortfall) },
-  });
+  // t.like(readLatest('published.reserve.metrics'), {
+  //   allocations: {
+  //     ATOM: { value: scale6(outcome.reserve.allocations.ATOM) },
+  //     // not part of product spec
+  //     Fee: { value: scale6(1.54) },
+  //   },
+  //   shortfallBalance: { value: scale6(outcome.reserve.shortfall) },
+  // });
```
Comment (Chris-Hibbert) [inline packages/vats/test/bootstrapTests/test-liquidation.js:558]: Why are these commented out? Should they be revived, or the reason documented?

**Review by Chris-Hibbert — APPROVED** [review]

File: packages/vats/test/bootstrapTests/test-liquidation.js (hunk 2 above)
Comment (turadg) [reply packages/vats/test/bootstrapTests/test-liquidation.js:558]: oops this file wasn't supposed to be included. it was renamed to `test-liquidation-1.js`. I've removed this in an amendment

File: packages/vats/test/bootstrapTests/test-liquidation-1.js (hunk 1 above)
Comment (turadg) [reply packages/vats/test/bootstrapTests/test-liquidation-1.js:253]: ok, changed all logs to trace, disabled

**Pull request merged**